packages/plugin-cosmos/src/index.ts

```diff
@@ -0,0 +1,15 @@
+export * from "./actions/transfer";
+
+import type { Plugin } from "@ai16z/eliza";
+import { transferAction } from "./actions/transfer";
+
+export const cosmosPlugin: Plugin = {
+    name: "cosmos",
+    description: "Cosmos blockchain integration plugin",
+    providers: [],
```

Comment:
we shoud create wallet provider to inform AI about our connected wallet state (mostly balance is related i think)